exercises/mongo-shell/intermediate-read.md

```diff
@@ -0,0 +1,146 @@
+## Comparison Operators
+
+These operators allow us to match a field's value relative to another value. 
+
+### Working with a single field
+
+**Exercise 1** :computer: 
+
+In the following query, we look at movies which had a runtime greater than 180 minutes (3 hours.)
+
+**query:**
+
+```javascript=
+db.movieDetails.find({"runtime": {"$gt":180}}, 
+    {"title": 1, "_id": 0})
+```
+
+**Exercise 2** :computer: 
+
+What about movies which ran for greater than or equal to 2 hours but less than 3 hours?
+
+**query:**
+```javascript=
+db.movieDetails.find({"runtime": {"$gte":120, "$lt": 180}}, 
+    {"title": 1, "_id": 0})
+```
+---
+
+### Working with multiple fields
+
+**Exercise 3** :computer: 
+
+List the movies which not only had a runtime greater than 3 hours but were also highly rated.
```

Comment:
What constitutes a high rating?
I also think it might be wise to get them used to words like `longer` and `higher` being associated with `greater`

```suggestion
List the movies which had a runtime longer than 3 hours and had a rating higher than 8
```